deploy/FlapperInit.sol

```diff
@@ -97,45 +92,35 @@ library FlapperInit {
 
     function initFlapperUniV2(
         DssInstance        memory dss,
-        FlapperInstance    memory flapperInstance,
+        address                   flapper,
         FlapperUniV2Config memory cfg
     ) internal {
-        FlapperUniV2Like flapper = FlapperUniV2Like(flapperInstance.flapper);
-        FlapperMomLike   mom     = FlapperMomLike(flapperInstance.mom);
+        FlapperUniV2Like flapper_ = FlapperUniV2Like(flapper);
```

Comment:
Maybe for reducing the amount of changes in the file is better the param is called `flapper_` and leave `flapper` as the one cast to the contract type. Also I'd think that would be more standard to other places (still pretty subjective).